OpenRA.Game/Traits/World/ActorMap.cs

```diff
@@ -192,6 +228,9 @@ public ActorMap(World world, ActorMapInfo info)
 			actorShouldBeRemoved = removeActorPosition.Contains;
 		}
 
+		/// <summary>
+		/// NOTE: Obsolete. Use GetActorsAt
```

Comment:
Exactly it is, that's why I marked the other 2 as obsolete. The problem is the return type. It's not the same to cast a IEnumerable<IActor> to IEnumerable<Actor> than IActor to Actor, I had to make changes to a whole lot of more files which would be inapproapriate for an already-big commit.

That's why I marked the other 2 as obsolete. A future PR should get rid of them.